src/compas/robots/resources/basic.py

```diff
@@ -55,12 +55,34 @@ def load_mesh(self, url):
 
 
 class DefaultMeshLoader(AbstractMeshLoader):
-    """Handles basic mesh loader tasks, mostly from local files."""
+    """Handles basic mesh loader tasks, mostly from local files.
 
-    def __init__(self):
+    Attributes
+    ----------
+    kwargs (optional): dict
+        Additional keyword arguments.
+    """
+
+    def __init__(self, **kwargs):
         super(DefaultMeshLoader, self).__init__()
+        self.attr = dict() or kwargs
```

Comment:
I think this should be reversed: `kwargs or dict()`. Now it basically always returns `kwargs`, because `dict()` is always `False` (because it's a valid but empty dictionary)

Reply:
Agree!